packages/create-request-form/src/routes/+page.svelte

```diff
@@ -0,0 +1,15 @@
+<script>
+  import { onMount } from 'svelte';
+  import { CreateInvoiceForm } from '$lib/index';
+  import { requestNetworkStore, initializeRequestNetwork } from '../stores/requestNetworkStore';
+  
+  onMount(() => {
+    initializeRequestNetwork();
+  });
+</script>
+
+{#if $requestNetworkStore}
+  <CreateInvoiceForm requestNetwork={$requestNetworkStore.requestNetwork} signer={$requestNetworkStore.signer} />
+{:else}
+  <p>Loading Request Network...</p>
```

Comment:
Does initialization take a long time? About how long?

Reply:
It's practically instant, and it asks user to sign the request so we can go on with the process, of course we can change this to a button and when user click that we ask for the sign, but at the end of the day it's on the builder how he will init the request network, this was only for testing purposes to see how we would pass the RN instance to the web component